package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@esm-bundle/autopublish-template",
-  "version": "20.0.0",
-  "description": "Repo to test esm-bundle's automatic publishing",
+  "name": "@esm-bundle/react-is",
+  "version": "16.13.1",
+  "description": "Repo to auto-esm-bundle react-is",
   "main": "index.js",
   "module": "esm/index.js",
   "type": "module",
   "scripts": {
     "test": "yarn build && concurrently -n w: 'yarn:test:*'",
-    "test:browser": "karma start karma.conf.cjs --single-run",
+    "test:browser": "karma start karma.conf.cjs",
     "debug:test:browser": "karma start karma.conf.cjs",
```

Comment:
This will only be a singleRun since that's what is the default in the config. `yarn debug:test:browser` should not be a single run.

Reply:
Reverted that change. I totally spaced that there were other scripts when I was testing.